# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../jahshaka/openlib-include.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- jahshaka/source/OpenLibraries/openimagelib/exr.cpp_orig	2006-10-19 23:14:17.823881500 +0200
+++ jahshaka/source/OpenLibraries/openimagelib/exr.cpp	2006-10-19 23:15:20.467796500 +0200
@@ -9,8 +9,8 @@
 #include <qimage.h>
 #include <qdatastream.h>
 
-#include <openpluginlib/pl/openpluginlib.hpp>
-#include <openpluginlib/pl/utf8_utils.hpp>
+#include <openpluginlib/openpluginlib.hpp>
+#include <openpluginlib/utf8_utils.hpp>
 
 #include <openimagelib/il/openimagelib_plugin.hpp>
 
--- jahshaka/source/OpenLibraries/openobjectlib/openobject.cpp_orig	2006-10-19 23:21:09.321598500 +0200
+++ jahshaka/source/OpenLibraries/openobjectlib/openobject.cpp	2006-10-19 23:21:22.838443250 +0200
@@ -28,8 +28,8 @@
 #include <openlibraries_global_config.hpp>
 #endif
 
-#include <openpluginlib/pl/openpluginlib.hpp>
-#include <openpluginlib/pl/utf8_utils.hpp>
+#include <openpluginlib/openpluginlib.hpp>
+#include <openpluginlib/utf8_utils.hpp>
 
 #include <openobjectlib/sg/openobjectlib_plugin.hpp>
 
--- jahshaka/source/OpenLibraries/openobjectlib/openobjectlib.cpp_orig	2006-10-19 23:20:07.269720500 +0200
+++ jahshaka/source/OpenLibraries/openobjectlib/openobjectlib.cpp	2006-10-19 23:20:19.410479250 +0200
@@ -18,8 +18,8 @@
 
 #ifdef NEWOPENOBJECTLIB
 
-#include <openpluginlib/pl/openpluginlib.hpp>
-#include <openpluginlib/pl/utf8_utils.hpp>
+#include <openpluginlib/openpluginlib.hpp>
+#include <openpluginlib/utf8_utils.hpp>
 
 namespace opl = olib::openpluginlib;
 
--- jahshaka/source/Jahshaka/JahDesktop/player/qtplayer.h_orig	2006-10-19 23:28:28.461043000 +0200
+++ jahshaka/source/Jahshaka/JahDesktop/player/qtplayer.h	2006-10-19 23:28:46.190151000 +0200
@@ -12,9 +12,9 @@
 #include <boost/thread/thread.hpp>
 #include <boost/thread/condition.hpp>
 
-#include <openpluginlib/pl/openpluginlib.hpp>
-#include <openpluginlib/pl/utf8_utils.hpp>
-#include <openpluginlib/pl/GL_utility.hpp>
+#include <openpluginlib/openpluginlib.hpp>
+#include <openpluginlib/utf8_utils.hpp>
+#include <openpluginlib/GL_utility.hpp>
 
 #include <openmedialib/ml/ml.hpp>
 
--- jahshaka/source/Jahshaka/JahModules/editing_mlt/audio.h_orig	2006-10-20 15:00:29.152089500 +0200
+++ jahshaka/source/Jahshaka/JahModules/editing_mlt/audio.h	2006-10-20 15:00:36.748564250 +0200
@@ -10,7 +10,7 @@
 #ifndef JAHNLE_MLTAUDIO_H
 #define JAHNLE_MLTAUDIO_H
 
-#include <openpluginlib/pl/timer.hpp>
+#include <openpluginlib/timer.hpp>
 #include <openmedialib/ml/ml.hpp>
 
 #include <iostream>
--- jahshaka/source/Jahshaka/JahModules/editing_mlt/gpupreview.cpp_orig	2006-10-20 15:03:31.435481500 +0200
+++ jahshaka/source/Jahshaka/JahModules/editing_mlt/gpupreview.cpp	2006-10-20 15:03:48.252532500 +0200
@@ -17,7 +17,7 @@
 #include <qtimer.h>
 #include <qmessagebox.h>
 
-#include <openpluginlib/pl/GL_utility.hpp>
+#include <openpluginlib/GL_utility.hpp>
 
 using namespace Mlt;
 using namespace jahnle;
--- jahshaka/source/Jahshaka/JahSource/jahcreate/jahcreate.cpp_orig	2006-10-20 15:07:07.476983250 +0200
+++ jahshaka/source/Jahshaka/JahSource/jahcreate/jahcreate.cpp	2006-10-20 15:07:18.133649250 +0200
@@ -37,7 +37,7 @@
 #include "MainEncoder.h"
 
 
-#include <openpluginlib/pl/openpluginlib.hpp>
+#include <openpluginlib/openpluginlib.hpp>
 #include <openobjectlib/sg/sg.hpp>
 #include <openobjectlib.h>
 
